scripts/openmc-make-test-data

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+
+"""
+Download ENDF/B-VII.1 ENDF and ACE files from NNDC and WMP files from GitHub and
+generate a full HDF5 library with incident neutron, incident photon, thermal
+scattering data, and windowed multipole data. This data is used for OpenMC's
+regression test suite.
+"""
+
+import glob
+import os
+from pathlib import Path
+import tarfile
+import tempfile
+from urllib.parse import urljoin
+import zipfile
+
+import openmc.data
+from openmc._utils import download
+
+base_ace = 'http://www.nndc.bnl.gov/endf/b7.1/aceFiles/'
+base_endf = 'http://www.nndc.bnl.gov/endf/b7.1/zips/'
+base_wmp = 'https://github.com/mit-crpg/WMP_Library/releases/download/v1.1/'
+files = [
+    (base_ace, 'ENDF-B-VII.1-neutron-293.6K.tar.gz', '9729a17eb62b75f285d8a7628ace1449'),
+    (base_ace, 'ENDF-B-VII.1-tsl.tar.gz', 'e17d827c92940a30f22f096d910ea186'),
+    (base_endf, 'ENDF-B-VII.1-neutrons.zip', 'e5d7f441fc4c92893322c24d1725e29c'),
+    (base_endf, 'ENDF-B-VII.1-photoat.zip', '5192f94e61f0b385cf536f448ffab4a4'),
+    (base_endf, 'ENDF-B-VII.1-atomic_relax.zip', 'fddb6035e7f2b6931e51a58fc754bd10'),
+    (base_wmp, 'WMP_Library_v1.1.tar.gz', '8523895928dd6ba63fba803e3a45d4f3')
+]
+
+
+def fix_zaid(table, old, new):
+    filename = os.path.join('tsl', table)
+    with open(filename, 'r') as fh:
+        text = fh.read()
+    text = text.replace(old, new, 1)
+    with open(filename, 'w') as fh:
+        fh.write(text)
+
+pwd = Path.cwd()
+output_dir = pwd / 'nndc_hdf5'
+os.makedirs('nndc_hdf5/photon', exist_ok=True)
+
+with tempfile.TemporaryDirectory() as tmpdir:
+    # Temporarily change dir
+    os.chdir(tmpdir)
```

Comment:
Optional: It may be worth it to put the logic following `chdir` into a `try` block, and put the `chdir` on line `164` into a `finally` block:

```
    os.chdir(    os.chdir(path)
    try:
       <Do stuff>
    finally:
        os.chdir(str(pwd)))
```

This ensures that even in case of some failure, the chdir gets changed back.